src/main/java/edu/kit/hci/soli/repository/SystemConfigurationRepository.java

```diff
@@ -14,6 +14,15 @@ public interface SystemConfigurationRepository extends Repository<SystemConfigur
      *
      * @return the {@link SystemConfiguration} instance.
      */
-    @Query("SELECT s FROM SystemConfiguration s")
+    @Query("SELECT s FROM SystemConfiguration s WHERE s.id = 1")
```

Comment:
SystemConfiguration is a singleton (this is enforced via the constraint on the id column) so this is not needed